projects/mongodb-backup-s3/resources/storage_and_access.tf

```diff
@@ -1,8 +1,18 @@
-module "paired_user" {
-    source = "../../../modules/paired_user"
+module "mongodb_s3_backup" {
+    source = "../../../modules/private_s3_bucket"
 
     bucket_name = "govuk-mongodb-backup-s3"
     environment = "${var.environment}"
     team        = "Infrastructure"
     username    = "govuk-mongodb-backup-s3"
 }
+
+module "mongodb_s3_daily_backpup" {
```

Comment:
and actually, the module here (and above) should be the name of the module rather than the bucket